<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test1"
           tests="2"
           failures="1"
           errors="0"
           skipped="0"
           time="3.195">
  <properties />
            <testcase name="TC02" time="3.177" classname="test_cases.LoginTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  test_cases.LoginTest.TC02(LoginTest.java:74)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
  org.testng.TestRunner.privateRun(TestRunner.java:766)
  org.testng.TestRunner.run(TestRunner.java:587)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
  org.testng.TestNG.runSuites(TestNG.java:1039)
  org.testng.TestNG.run(TestNG.java:1007)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="TC01" time="0.018" classname="test_cases.LoginTest" />
      
</testsuite>
